src/CourseDashboard.cpp

```diff
@@ -89,9 +89,9 @@ void CourseDashboard::updateUser(User & user)
     }
 }
 
-void CourseDashboard::saveToFile()
+void CourseDashboard::saveToFile(std::string fileName)
 {
```

Comment:
Was it tested that filename with space works? If not we need include it in tests

Reply:
Yes, filename with space works